app/models/allocation.rb

```diff
@@ -85,10 +85,12 @@ def allocate_all_claims_or_none!(claims)
 
   def allocate_or_error_claim!(claim)
     if claim.case_workers.exists?
-      errors.add(:base,"Claim #{claim.case_number} has already been allocated to #{claim.case_workers.first.name}")
+      errors.add(:base, "Claim #{claim.case_number} has already been allocated to #{claim.case_workers.first.name}")
     else
       allocate_claim! claim
     end
+  rescue => ex
```

Comment:
In this case we don't really care about the exception, just any exception will do, as it is going to rollback the whole transaction. We are not recovering from it so I don't see the point of specifying for example StateMachines::InvalidTransition (that's the one being raised) when some other might be raised down the line.